core/src/main/java/bio/terra/pearl/core/service/workflow/EnrollmentService.java

```diff
@@ -165,9 +169,10 @@ public HubResponse<Enrollee> enrollAsProxy(EnvironmentName envName, String study
     public HubResponse<Enrollee> enrollGovernedUser(EnvironmentName envName, String studyShortcode, Enrollee governingEnrollee,
                                                     ParticipantUser proxyUser, PortalParticipantUser proxyPpUser,
                                                     UUID preEnrollResponseId, String governedUserName) {
+        ParticipantUser governedUserParticipantUserOpt = participantUserService.findOne(governedUserName, envName).orElse(null);
```

Comment:
Since in normal case governedUserName is generated from the email concatenated with a random string, this will be null 99.9% of the times. The only real usecase is in time of population, to avoid recreating the same user again.